tests/bearlib/aspects/DecoratorsTest.py

```diff
@@ -16,11 +17,20 @@ class RunDecoratedBear(LocalBear):
         remove_unreachable_code=get_aspect('UnreachableCode'),
         minimum_clone_tokens=get_aspect('Clone').min_clone_tokens,
     )
+    @map_ambiguous_setting_to_aspect(
+        use_spaces=(get_aspect('Indentation').indent_type,
+                    [('space', True), ('tab', False)]),
+        blanklines_at_EOF=(get_aspect('NewlineAtEOF').newline_at_EOF,
```

Comment:
blanklines != newline *token*.  They are orthogonal.  They can not be mapped sensibly, and doing it in your test code makes the test logic confusing.

A newline token at EOL is terminating the last line with a newline token.

The *meaning* of that newline token varies between context and GUI editors, but it is not a blank line.